### DISCUSSION TITLE
disabling virtual text

### DISCUSSION THREAD

**Discussion by NoamMuallem** [LazyVim/LazyVim#939] [General]
Hi, I want to add the plugin lsp_lines.nvim, in the config snippet it shows this:
```
-- Disable virtual_text since it's redundant due to lsp_lines.
vim.diagnostic.config({
  virtual_text = false,
})
```
I added this to my config, right under where I require the plugin, but the regular virtual text is still showing.
did someone try this?

**Comment by dpetka2001** [comment]
You should add this under your `nvim-lspconfig` spec, since that's responsible for the virtual text.

**Reply by dpetka2001** [reply]
It's described in the [docs](https://www.lazyvim.org/plugins/lsp#nvim-lspconfig)

**Reply by Aljendro** [reply]
```
return {
  "neovim/nvim-lspconfig",
  opts = function(_, opts)
    opts.diagnostics.virtual_text = false;
  end,
}
```
add it to the .config/nvim/lua/plugins directory as a lua file for anyone who is confused.

**Reply by luispinto23** [reply]
> ```
> return {
>   "neovim/nvim-lspconfig",
>   opts = function(_, opts)
>     opts.diagnostics.virtual_text = false;
>   end,
> }
> ```
> 
> add it to the .config/nvim/lua/plugins directory as a lua file for anyone who is confused.

I've done this but am still getting the virtual text. I was trying to disable it to use the tiny-inline-diagnostics plugin but it's not really disabling them, I still got both as you can see in the image (the config error is on purpose to illustrate the problem)
![image](https://github.com/user-attachments/assets/18e8d30f-cf1b-469b-a077-19092ba39b59)

**Comment by marcinjahn** [comment]
Could you show how you even added lsp_lines to LazyVim?
